### DISCUSSION TITLE
can i join the team sir?

### DISCUSSION THREAD

**Discussion by mozzaru** [ROM-builders/temporary#18116] [General]
Tree : https://github.com/Mozzaru/android_device_xiaomi_markw

Kernel : https://github.com/Mozzaru/android_kernel_xiaomi_markw_new

Vendor: https://github.com/Mozzaru/proprietary_vendor_xiaomi

Local manifest:
https://github.com/Mozzaru/local_manifest

build and boot results:
https://sourceforge.net/projects/rom-builders/files/markw/

https://t.me/vanguard78
https://t.me/bolanng77

**Comment by Apon77** [comment]
please follow #learn instruction in telegram

**Reply by mozzaru** [reply]
@Apon77 ok

**Reply by mozzaru** [reply]
![Screenshot_2023-02-16-10-57-13-36_4d38fce200f96aeac5e860e739312e76](https://user-images.githubusercontent.com/100791931/219265171-e611c953-0838-432e-9494-26302ef183e9.jpg)
@Apon77 sir
